test/e2e/project.mk

```
@@ -0,0 +1,34 @@
+# Project specific values
+OPERATOR_NAME?=configuration-anomaly-detection
+
+E2E_SUITE_IMAGE_REGISTRY?=quay.io
+E2E_SUITE_IMAGE_REPOSITORY?=app-sre
+E2E_SUITE_IMAGE_NAME?=$(OPERATOR_NAME)-test-
```

```suggestion
E2E_SUITE_IMAGE_NAME?=$(OPERATOR_NAME)-e2e
```

or something, whatever you end up making the image name in app-interface